RedHatInsights/insights-inventory-frontend
# fix(InventoryTable): RHINENG-9872 - Make download more reliable

This changes the export check for downloads to use the `/status` endpoint of the exports to check for the completion. The download check now also recognises "failed" exports and will abort the download (check) and trigger an error notification. And the filename provided in the content disposition header is used for the downloaded filed if provided.

**How to test:** 

1) Ensure the export feature is enabled for the inventory
2) Visit the Systems page
3) Click the "export button" and choose either CSV or JSON to download
4) Verify the export was properly triggered and the export was downloaded on success
4.1) There is no easy way to verify the error handling. But exports on accounts with many systems may still fail at the moment.
